src/core/PointLight.ts

```diff
@@ -64,6 +45,10 @@ export default class PointLight extends LightBase {
 
         if (!this.isConnected) return
 
+        if (modifiedProps.distance) this._forwardProp('distance', this.three)
+        if (modifiedProps.decay) this._forwardProp('decay', this.three)
+        if (modifiedProps.castShadow) this._forwardProp('castShadow', this.three)
```

Comment:
The above instances of `(self: any) => self.three` were replaced with these lines inside of the `updated()` method.

Reply:
Overall, I believe this is also easier to understand.